core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -178,9 +186,22 @@ public void onSurfaceHolderReady(SurfaceHolder surfaceHolder) {
                 exoPlayer.loadVideo(surfaceHolder, drmSessionCreator, uri, contentType, forwarder, mediaCodecSelector);
             }
         };
+
+        assertPlayerViewIsAttached();
+        forceSurfaceToBeCreated();
         surfaceHolderRequester.requestSurfaceHolder(onSurfaceReadyCallback);
     }
 
+    private void assertPlayerViewIsAttached() {
+        if (playerView == null) {
+            throw new IllegalStateException("A PlayerView must be attached in order to loadVideo");
```

Comment:
yep it was added so that they match, my end goal is to allow the players to play without a surface to enable audio only playback of videos